### ISSUE TITLE
Is multiprocessing(threading) supported now?

### ISSUE THREAD

**Issue by 92ada** [ZoomerAnalytics/xlwings#1008]
#### OS
macOS 
#### Versions of xlwings, Excel and Python
newest xlwings, Office 365, Python3.6

**Comment by fzumstein** [comment]
Yes, threading is now supported, see: http://docs.xlwings.org/en/stable/threading.html but multiprocessing seems to only work on Windows. I will update the docs accordingly.

**Comment by fzumstein** [comment]
Reference to the yet unpublished docs: https://github.com/ZoomerAnalytics/xlwings/blob/master/docs/threading_and_multiprocessing.rst

**Comment by 92ada** [comment]
Thanks ;)
